src/app/information/information.component.ts

```diff
@@ -11,51 +13,38 @@ import { changeInfoPropertiesAction } from '@app/episode/info/store/info.actions
   styleUrls: ['./information.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class InformationComponent implements OnInit {
-  episodeInfo$ = this.store.select(selectEpisodeInfo).pipe(
-    tap((value) => console.log('Episode Info', value)),
-    startWith(emptyEpisodeInfo()),
-    tap((value) => console.log('Episode Info', value)),
-  );
+export class InformationComponent implements OnInit, OnDestroy {
+  /**
+   * Auto-save fields might fail, this flag will determine if there is something unsaved
+   */
+  isPendingUpdates = false;
+  /**
+   * UI Model for binding episode info
+   */
+  episodeInfo: EpisodeInfo;
```

Comment:
This will serve two purposes:
- UI will bind and update this object, the UI already waits it to be set on the `*ngIf`
- On **blur**, the whole page will be persisted to Redux (and saved) instead of each keystroke

Because we are always keeping the model updated, there is no need to re-update it after being saved.